DMFT/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.data.mongodb.host=localhost
+spring.data.mongodb.port=27017
+spring.data.mongodb.authentication-database=admin
+spring.data.mongodb.username=myUserAdmin
+spring.data.mongodb.password=admin
```

Comment:
We don't want to keep secrets in source code.  Extract to `.env` file so that the application uses environment variables for secrets.

Reply:
I'll make changes and add this file to .gitignor